resources/local/chainspec.toml.in

```diff
@@ -48,7 +48,7 @@ round_seigniorage_rate = [15_959, 6_204_824_582_392]
 # It is the fraction of validators that would need to equivocate to make two honest nodes see two conflicting blocks as
 # finalized: A higher value F makes it safer to rely on finalized blocks.  It also makes it more difficult to finalize
 # blocks, however, and requires strictly more than (F + 1)/2 validators to be working correctly.
-finality_threshold_fraction = [1, 10]
+finality_threshold_fraction = [1, 3]
```

Comment:
We should/could modify this in `nctl` though.

Reply:
we could switch it only for nctl, but if mainnet plans to be run with 1/3 we should probably default to that. I'll let others chime in though , I'll bring it up in standup today

Reply:
I agree as general rule but some tests will be easier to run with different values. :+1:

Reply:
After discussing in standup, we decided to proceed with this as 1/3.